pkg/executor/build.go

```diff
@@ -167,6 +168,10 @@ func (s *stageBuilder) build(opts *config.KanikoOptions) error {
 			return err
 		}
 		files := command.FilesToSnapshot()
+		if command.Name() == "volume" {
```

Comment:
I think `cmd.Name()` should work here, that way you won't need to add the function to every command.

Let's also add `volume` to `constants.go`